Makefile

```diff
@@ -65,18 +65,24 @@ ifeq ($(PULL_ARTIFACTS),no)
 endif
 
 build:
-ifeq ($(PULL_ARTIFACTS),yes)
-	python3 -u ./py/confluent/build/download_artifacts/download_artifacts.py
-endif
-
 ifeq ($(SKIP_TESTS),yes)
 	mvn -DskipTests=true install
 else
 	mvn install
 endif
 
-install: build
+newbuild:
+	git clone git@github.com:confluentinc/download-artifacts.git
+	python3 -u ./download-artifacts/py/confluent/build/download_artifacts/download_artifacts.py
+	rm -rf ./download-artifacts
+
+install:
+ifeq ($(PULL_ARTIFACTS),yes) 
+	make newbuild
+else 
+	make build
```

Comment:
i'm not sure it's actually safe to get rid of `create_archive.sh` fully here for the new build  path.  You can move the commands into the Makefeile (e.g. as has ben done for many of the  services, such as https://github.com/confluentinc/schema-registry/blob/master/debian/Makefile#L69), but that file sometimes does some rearrangement of artifacts (e.g. dealing with  PREFIX vs not), uses `install` to set user/group/permissions properly, filters some extra  stuff (looks like `README.rpm` commonly, though we should just filter that out  earlier if we  want).